### PR TITLE
16994 cypress viewport updater documentation

### PR DESCRIPTION
**Pull request by holdenhinkle**
Regarding ticket https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/16994

This PR is to add documentation for the Cypress Viewport Updater Bot.

### CODE REVIEW

**Comment by holdenhinkle** [comment]
PRs for the Cypress Viewport Updater Bot:  
- Part 1 - https://github.com/department-of-veterans-affairs/vets-api/pull/5800
- Part 1.5 - https://github.com/department-of-veterans-affairs/vets-api/pull/5840
- Part 2 - https://github.com/department-of-veterans-affairs/vets-api/pull/5841
- Part 3 - https://github.com/department-of-veterans-affairs/vets-api/pull/5842
- Part 4 - https://github.com/department-of-veterans-affairs/vets-api/pull/5845
- Part 5 - https://github.com/department-of-veterans-affairs/vets-api/pull/5846
- Part 6 - https://github.com/department-of-veterans-affairs/vets-api/pull/5847
- Part 7 - https://github.com/department-of-veterans-affairs/vets-api/pull/5848

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Cypress Viewport Updater Bot
+
+The Automatic Cypress Viewport Updater updates the following files in `vets-website` each month with recent data from Google Analytics:
+
+- `config/cypress.json`
+- `src/platform/testing/e2e/cypress/support/commands/viewportPreset.js`
+
+The updater:
+
+- Requests two Google Analytics reports from the `ga.gov` organization:
```
Comment (1Copenut) [inline platform/testing/end-to-end/automatic-cypress-viewport-updater.md:10]: Should this be the `va.gov` organization?

**Review by 1Copenut — CHANGES_REQUESTED** [review]
Just one quick comment/change request. Looks good!

File: platform/testing/end-to-end/automatic-cypress-viewport-updater.md (hunk 1 above)
Comment (holdenhinkle) [reply platform/testing/end-to-end/automatic-cypress-viewport-updater.md]: Um... yeah!?! haha, thanks for catching that typo!

**Review by 1Copenut — APPROVED** [review]
👍 LGTM!

**Pull request merged**